packages/mobile/src/components/comments/CommentOverflowMenu.tsx

```diff
@@ -249,7 +250,10 @@ export const CommentOverflowMenu = (props: CommentOverflowMenuProps) => {
 
       <Portal hostName='DrawerPortal'>
         {isVisible ? (
-          <CommentSectionProvider entityId={entityId}>
+          <CommentSectionProvider
+            entityId={entityId}
+            lineupActions={tracksActions}
```

Comment:
We could probably make lineupActions optional, and not pass it here. Since I don't think anything in the overflow menu needs to play tracks. But if it's less complicated to keep it consistent then that's fine too

Reply:
Yeah thats fair, I think I will go ahead and do that